tests/ext/sandbox-prehook/access_modifier_method_access_hook.phpt

```diff
@@ -45,13 +45,13 @@ class Bar
     }
 }
 
-dd_trace_method('Foo', 'aPublic', ['prehook' => function() {
+DDTrace\trace_method('Foo', 'aPublic', ['prehook' => function() {
```

Comment:
Missing leading `\`.

Reply:
Yes, this is intentional for the phpt tests since they are all in the root namespace anyway (with the exception of the namespace test). So I just kept the calling reference the same as the original. :)

Reply:
👍  sounds good them, resolving this and the next ones